app/products/calls/components/current_call_bar/current_call_bar.tsx

```diff
@@ -123,13 +124,10 @@ const CurrentCallBar = ({
                 visible: Platform.OS === 'android',
             },
         };
-        goToScreen('Call', 'Call', {}, options);
+        goToScreen(Screens.CALL, 'Call', {}, options);
```

Comment:
leaving the comment so that it can be addressed at a later time.

Let's localize the title `Call`

Reply:
Sure thing, done: 9066a6b69f4944e04a9ab0853bf3dd917a745612
We have a ticket for the rest of the localizations, not to worry :)  https://mattermost.atlassian.net/browse/MM-45945